tests/base/convergence_table_05.cc

```diff
@@ -0,0 +1,42 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2010 - 2015 by the deal.II authors
```

Comment:
I can never remember - do these dates get auto-corrected by a script before we do a release?

Reply:
I think so, yes.

Reply:
Yes.